models/event.js

```diff
@@ -0,0 +1,83 @@
+const cozy = require('../cozyclient');
+const DOCTYPE = 'io.cozy.events';
+
+module.exports = {
+  displayName: 'Event',
+  doctype: DOCTYPE,
+  all (callback) {
+    cozy.data.defineIndex(DOCTYPE, [])
```

Comment:
I tried it with the fixture and it worked, but I'll try with your implementation 😉